ES/ESmy022.xml

```diff
@@ -155,155 +133,24 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   </physDesc>
                   <history>
                      <origin>
-
-
-                        <origDate notBefore="1850" notAfter="1950">Mid-19th to mid-20th century</origDate>
-                        for <ref target="#p1">Unit 1</ref> and
-                        <origDate notBefore="1950" notAfter="2009">Mid-20th to the beginning of the 21st century</origDate> for
-                        <ref target="#p2">Unit 2</ref>. Produced in the recent time, <ref target="#p2">Unit 2</ref> was unlikely ever a circulation unit;
-                        it was introduced into the binding of <ref target="#p1">Unit 1</ref> before the manuscript
+                        <origDate notBefore="1850" notAfter="1950">Mid-19th to mid-20th century.</origDate>
+                        for <ref target="#p2">Unit 2</ref>, and
+                        <origDate notBefore="1950" notAfter="2009">mid-20th to the beginning of the 21st century</origDate> for
+                        <ref target="#p1">Unit 1</ref>. Produced in the recent time, <ref target="#p1">Unit 1</ref> was unlikely
+                        ever a circulation unit. It was introduced into the binding of <ref target="#p1">Unit 2</ref> before the manuscript
                         was digitized in <date>2010</date>.
-                        The names of its commissioners are listed in the concluding formula on <locus from="5v" to="6r"/>,
-                        ʾabuna Zarʾa Bǝruk, ʾabuna Gabra Yoḥannǝs, ʾabuna
-                        ʾƎqʷǝba Yoḥannǝs, Walatta Ṣādǝqan, Walatta Krǝstos,
-                        Walatta Gabrǝʾel and ʾabuna ʾAsaba Māryām.
+                        <ref target="#p1">Unit 1</ref> was commissioned by a group of individuals listed
+                        in the concluding formula on <locus from="5v" to="6r"/>,
+                        <persName ref="PRS13186ZaraBuruk">ʾabuna Zarʾa Bǝruk</persName>,
```

Comment:
if you add role="donor" here you do not have to repeat the list below.

Reply:
Should I remove it or it can stay?

Reply:
it can stay, but as a rule, if you assign anywhere in the description roles (donor/owner/scribe) these are read by the system and there is no necessity to repeat them below. no error here.